src/sidebar/components/PaginationNavigation.tsx

```diff
@@ -1,24 +1,55 @@
-import { LabeledButton } from '@hypothesis/frontend-shared';
+import {
+  ButtonBase,
+  ArrowLeftIcon,
+  ArrowRightIcon,
+} from '@hypothesis/frontend-shared/lib/next';
+import type { PresentationalProps } from '@hypothesis/frontend-shared/lib/types';
+import type { ButtonCommonProps } from '@hypothesis/frontend-shared/lib/components/input/ButtonBase';
 import classnames from 'classnames';
+import type { JSX } from 'preact';
 
 import { pageNumberOptions } from '../util/pagination';
 
+type NavigationButtonProps = PresentationalProps &
+  ButtonCommonProps &
+  Omit<JSX.HTMLAttributes<HTMLButtonElement>, 'icon' | 'size'>;
+
 /**
- * @typedef PaginationNavigationProps
- * @prop {number} currentPage - The currently-visible page of results. Pages
- *   start at 1 (not 0).
- * @prop {(page: number) => void} onChangePage - Callback for changing page
- * @prop {number} totalPages
+ * Style a navigation button.
+ *
+ * @param {PresentationalProps & ButtonCommonProps & HTMLButtonAttributes} props
  */
+function NavigationButton({ ...buttonProps }: NavigationButtonProps) {
+  return (
+    <ButtonBase
+      classes={classnames(
+        'px-3.5 py-2.5 gap-x-1 font-semibold',
+        // These colors are the same as the "dark" variant of IconButton
+        'text-grey-7 bg-grey-2 enabled:hover:text-grey-9 enabled:hover:bg-grey-3 disabled:text-grey-5 aria-pressed:bg-grey-3 aria-expanded:bg-grey-3'
```

Comment:
I think it would help readability slightly to split the styles for enabled/disabled states onto a separate line. Also, when does `aria-expanded` get used for this button, since it doesn't have an associated popup menu?

Reply:
Broken into two lines for readability.

I brought all `aria-` classes over as a reminder that, if and when this button's style gets extracted as a pattern (e.g. as a button `variant` or somesuch) that those states need to be handled.